dc/council/code/titles/24/sections/24-1102.xml

```diff
@@ -3,13 +3,14 @@
   <num>24-1102</num>
   <heading>Authority to enter into and execute Compact.</heading>
   <text>The Mayor is authorized to enter into and execute on behalf of the District a compact with any state or states legally joining therein in the form substantially as follows:</text>
+  <text>
+    <center>
+        <strong>THE INTERSTATE COMPACT FOR JUVENILES</strong>
+    </center>
+  </text>
   <para>
```

Comment:
So, really, these are `<include>`s with a `<document>` and one or more `<section>`s. The `<para>` with a `<prefix>` and a `<num>` is to capture the structure of the article, _for now_. Later, the intention is to add the aforementioned elements. `<text>` elements don't quite capture the structure we need here.